src/liboqs.pc.in

```diff
@@ -0,0 +1,10 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+libdir=${prefix}/lib
+includedir=${prefix}/include
+
+Name: @PROJECT_NAME@
+Description: Library for quantum-safe cryptographic algorithms
+Version: @OQS_VERSION_TEXT@
+Requires.private: openssl
```

Comment:
`liboqs` often requires openssl, but this is optional

Reply:
I'm thinking this would be best solved with an internal variable that is set to "openssl" if `OQS_USE_OPENSSL` is enabled. Is there some cmake variable naming convention in this project, e.g. `_oqs_pkgconfig_requires_private`?